airflow/bin/cli.py

```diff
@@ -401,18 +401,23 @@ def webserver(args):
         app.run(debug=True, port=args.port, host=args.hostname)
     else:
         pid, stdout, stderr, log_file = setup_locations("webserver", pid=args.pid)
+        secure_params = True if args.ssl_certfile and args.ssl_keyfile else False
```

Comment:
can't this just be:

    secure_params = args.ssl_certfile and args.ssl_keyfile